features/user_can_pay_for_order.feature

```diff
@@ -0,0 +1,71 @@
+Feature: User can pay for order
+  As a visitor
+  In order to complete my purchase
+  I need to fill in a stripe payment form
+
+  @javascript
+  Scenario: User can fill in stripe form
+    Given I visit the cart page
+    When I click "Pay with Card" button
+    Then I fill in stripe form field "Email" with "random@random.com"
+    And I fill in stripe form field "Card number" with "4242 4242 4242 4242"
+    And I fill in stripe form field "Expiry" with "12/2021"
+    And I fill in stripe form field "CVC" with "123"
+    And I submit the stripe form
+    Then I should be on the "cart" page
```

Comment:
Probably best to change the 'Then' to 'And' because you're already writing 'Then' on line 10.

Reply:
👍